app/Lib/lang.php

```diff
@@ -1460,8 +1460,8 @@
   'fd.telephone_number.description' => 'Description',
   // This one is for rendering into a telephone number string (eg: 555 1212 x279)
   'fd.telephone.ext' => 'x',
-  'fd.cert.subject' => 'Subject',
-  'fd.cert.issuer' => 'Issuer',
+  'fd.cert.subject' => 'Cert Subject DN',
```

Comment:
Is there a reason why we need the abbreviation here instead of Certificate?

Reply:
Done